pkgs/coverage/lib/src/coverage_options.dart

```diff
@@ -0,0 +1,78 @@
+import 'dart:io';
+import 'package:cli_config/cli_config.dart';
+
+class CoverageOptions {
+  const CoverageOptions({
+    required this.output,
+    required this.scopeOutput,
+    required this.functionCoverage,
+    required this.branchCoverage,
+    required this.packagePath,
+    this.packageName,
+    required this.testScript,
+  });
+
+  factory CoverageOptions.fromConfig(
+      Config options, CoverageOptions defaultOptions) {
+    return CoverageOptions(
+      output: options.optionalString('out') ?? defaultOptions.output,
+      scopeOutput: options.optionalStringList('scope_output') ??
+          defaultOptions.scopeOutput,
+      functionCoverage: options.optionalBool('function_coverage') ??
+          defaultOptions.functionCoverage,
+      branchCoverage: options.optionalBool('branch_coverage') ??
+          defaultOptions.branchCoverage,
+      packagePath:
+          options.optionalString('package') ?? defaultOptions.packagePath,
```

Comment:
Can you evaluate the path to an absolute path here? The path should be relative to the config file path. Same for the output directory.